src/sh/python-lint/poetry-check.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+POETRY_SUBDIRECTORY=./src/python/hooks
+
+poetry --version >/dev/null 2>&1 || {
+	echo "Poetry is not installed. Please install poetry to continue."
+	exit 1
+}
```

Comment:
Per eng call how about a `poetry install` command here that installs the python hook project dependencies into the current environment?

Reply:
Captured at https://github.com/econia-labs/econia-v5/pull/42#discussion_r1544847872